test/integration/generated/bodyString/src/operations/string.ts

```diff
@@ -365,19 +387,21 @@ export class String {
     stringBody: Uint8Array,
     options?: coreHttp.RequestOptionsBase,
     callback?: coreHttp.ServiceCallback<any>
```

Comment:
Do we still need to support the callback pattern? The other day @ramya-rao-a mentioned that we may no longer need to. Any thoughts?
/cc @chradek @daviwil

Reply:
Since it's easy to interact with promises via callbacks without async/await, I'd say the old-style callback pattern shouldn't be needed any longer.  However, it'd be good to get @bterlson's opinion on that from an API guidelines perspective.

Reply:
Agreed that I think we should be able to drop callback support. The only reason I see to continue supporting it is to help users migrate their existing code, but it's easy to 'callbackify' a promise if needed.

Reply:
No callbacks if we have promises. Get rid of it if possible :)

Reply:
Cool! I will get rid of them in this PR :) so this
![image](https://user-images.githubusercontent.com/21109913/72643089-bf565980-3922-11ea-8c47-e7d9d8443e14.png)


Will turn into this
![image](https://user-images.githubusercontent.com/21109913/72643119-cc734880-3922-11ea-9cd0-407214996bca.png)

Reply:
Remove the callback parameter